Lib/multiprocessing/pool.py

```diff
@@ -149,8 +149,9 @@ class Pool(object):
     '''
     _wrap_exception = True
 
-    def Process(self, *args, **kwds):
-        return self._ctx.Process(*args, **kwds)
+    @staticmethod
```

Comment:
Why do you use `classmethod` at times and `staticmethod` at others? I'd rather settle on `classmethod`.

Reply:
The previous code did it as well (mix them). Also most of the class before the patch is @staticmethod . The spirit of why this class is constructed this way I believe is to avoid the circular reference at all costs (so do everything as staticmethod as possible and pass the function everything that is needed) yet still allow inheritance where needed (which is why _terminate_pool is a classmethod so that ThreadPool can override _help_stuff_finish and still work ok).

So If so, I guess I should make everything staticmethod, yes ?

Reply:
Well, `classmethod` shouldn't create any circular references either. But if `staticmethod` is being used elsewhere then ok.